### ISSUE TITLE
BatchNorm FIX still needed?

### ISSUE THREAD

**Issue by dsantiago** [hunglc007/tensorflow-yolov4-tflite#155]
Isn't this BatchNorm FIX, already fixed in new releases of TF 2? Looks like i think:

https://github.com/hunglc007/tensorflow-yolov4-tflite/blob/dafb42f3f2c4efacaa854613a4fe0300d39edcba/core/common.py#L6-L17

Isn't this BatchNorm FIX, already fixed in new releases of TF 2? Looks like i think:

https://github.com/tensorflow/tensorflow/blob/e925db6faef1067b97b9bc76238ba32a4d29b83b/tensorflow/python/keras/layers/normalization_v2.py#L207-L247
